evm/utils.js

```diff
@@ -636,18 +636,27 @@ const mainProcessor = async (options, processCommand, save = true, catchErr = fa
         throw new Error('Environment was not provided');
     }
 
-    if (!options.chainName && !options.chainNames) {
-        throw new Error('Chain names were not provided');
-    }
-
     printInfo('Environment', options.env);
 
     const config = loadConfig(options.env);
-    let chains = options.chainName ? [options.chainName] : options.chainNames.split(',');
```

Comment:
do any scripts still use `chainName`? switch to the generic chainNames option to simplify